examples/docker-hello-world/docker_hello_world.md

```diff
@@ -11,6 +11,8 @@ For all the supported key value properties, see [Code to Cloud specification](ht
 ::: code Cloud.toml :::
 
 Execute the `bal build` command to build the Ballerina package. Code to Cloud generates only one container per package.
+>**Note:** For Mac users, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is necessary because the Ballerina base image doesn't yet support the M1 chip.
```

Comment:
Should this be
```suggestion
>**Note:** For Mac users with Apple Silicon chips, run `export DOCKER_DEFAULT_PLATFORM=linux/amd64` before building the image. This is necessary because the Ballerina base image doesn't yet support the M1 chip.
```
? 

 I think this is not related to Mac users having intel chips, right?